docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -0,0 +1,78 @@
+# Upgrade to 8.0.0
+
+## New dependencies
+
+Depending on the used OS, run the corresponding command.
+
+### FreeBSD
+
+```sh
+pkg install p5-Plack-Middleware-ReverseProxy
+```
+
+### Debian / Ubuntu
+
+```sh
+sudo apt-get install libplack-middleware-reverseproxy-perl
+```
+
+### Centos
+
+```sh
+sudo cpanm Plack::Middleware::ReverseProxy
+```
+
+
+## Upgrading init scripts
+
+Go to the share folder:
+```
+cd `perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")'`
+```
+
+And then install the `zm-rpcapi` daemon (`zm_rpcapi` on FreeBSD).
```

Comment:
The comment is not needed. There is a risk that you start doing that without seeing the commands below.